custom_components/solis/__init__.py

```diff
@@ -8,10 +8,10 @@
 import voluptuous as vol
 from homeassistant.config_entries import ConfigEntry, SOURCE_IMPORT
 from homeassistant.const import Platform, ATTR_NAME, CONF_NAME
-from homeassistant.core import HomeAssistant
+from homeassistant.core import HomeAssistant, ServiceCall
```

Comment:
FYI, `ServiceCall` is not used in this file so you can remove this too.

```suggestion
from homeassistant.core import HomeAssistant
```